slides.Rmd

```diff
@@ -547,46 +590,80 @@ baby6 <- read_csv("data/Baby6.csv")
 baby_joined <- baby5 %>%
   left_join(baby6, by = c("FAMILYID", "DOB"))
 ```
-  
 
 
-## Data Visulisation 
+## END BLOCK 3
+
+
+## END BLOCK 4
+
+
+## Data Visulisation - ggplot2
+
+```{r, echo = FALSE}
+# TRAINERS NOTES
+# Maybe we need a slide here giving a bit of background to ggplot2? 
+```
 
 A basic line graph can be created using the following code:
 
 ```{r,  eval=FALSE,echo=TRUE}
+install.packages("ggplot2")
+library(ggplot2)
+
+#Define data
 x <- c(1, 3, 2, 4)
-# Plot the data
-plot(x, type = "l")
-```
+y <- c(4, 8, 7, 11)
 
- Note that `type = "l"` simply means to plot a line.
+# Create a data frame
+df <- as.data.frame(cbind(x, y))
 
+# Plot the data
+line_plot <- ggplot(df, aes(x, y)) + 
+  geom_line() +
+  geom_point()
 
-```{r, echo=FALSE}
-x <- c(1, 3, 2, 4)
-par(mar = c(10, 4, 4, 2))
-plot(x, type = "l")
+# View plot
+line_plot
```

Comment:
It would be good to show the plot in the slide, maybe the next slide is view plot then the plot rendered?